monkestation/code/game/objects/structures/crates_lockers/closets/secure/clown.dm

```diff
@@ -0,0 +1,28 @@
+/obj/structure/closet/secure_closet/clown
+	name = "\proper clown locker"
+	icon = 'monkestation/icons/obj/closet.dmi'
+	max_integrity = 325 //About 14 hits with the fire axe
+	icon_state = "clown"
+
+/obj/structure/closet/secure_closet/clown/PopulateContents()
+	..()
+	new	/obj/item/megaphone/clown(src)
+	new	/obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter(src)
+	new /obj/item/pneumatic_cannon/pie(src)
+	new /obj/item/food/pie/cream(src)
+	new /obj/item/storage/crayons(src)
+	new /obj/item/soundsynth(src)
+
+/obj/structure/closet/secure_closet/clown/togglelock(mob/living/user, silent)
+	if(secure && !broken)
+		if(user.mind.assigned_role == "Clown")
```

Comment:
You have to make sure they even have a mind.
The possibility is so tiny that they won't, but it's better we handle it now.

```suggestion
		if(user?.mind.assigned_role == "Clown")
```

Reply:
what would be the consequence if the clown doesn't have a mind?

Reply:
It might come up with an error or something if it tries to check for something that isn't there